### ISSUE TITLE
[issue]: "Show All Devices" doesn't work with internal devices

### ISSUE THREAD

**Issue by marwenius** [ventoy/Ventoy#1698]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.77

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

Both

### Partition Style

MBR

### Disk Capacity

512 GB

### Disk Manufacturer

Western Digital

### What happened?

When I click on "Show All Devices" no internal drives are shown to me, even if I click on update.

With the previous version (1.0.76) this worked.

It does not matter whether I use "Ventoy2Disk.exe" or "Ventoy2Disk_X64.exe".

![grafik](https://user-images.githubusercontent.com/108332626/176143533-f145efc3-1cc2-48f6-aff2-d657f3da7ca0.png)

![grafik](https://user-images.githubusercontent.com/108332626/176143905-4c8f7bef-f9cf-4529-82fe-8d5e983860a5.png)

The drives in question:
- Samsung 980 NVMe 1 TB
- WD SN520 NVMe 512 GB (the drive I want to update)

System:
- Lenovo ThinkPad T480 i7-8650U
- Windows 10 Pro 21H2 (x64)
- Secure Boot enabled

**Comment by ventoy** [comment]
Please run Ventoy2Disk.exe 1.0.77 enable Show All Device and upload the log.txt here.
Then download 1.0.76 and do the same operation and also upload the log.txt here.

**Comment by marwenius** [comment]
[v1076_log.txt](https://github.com/ventoy/Ventoy/files/9001735/v1076_log.txt)

[v1077_log.txt](https://github.com/ventoy/Ventoy/files/9001734/v1077_log.txt)

**Comment by ventoy** [comment]
```
1.0.76 log
[2022/06/28 15:45:40.765] Create file Handle:00000000000000DC \\.\PhysicalDrive0 status:0
[2022/06/28 15:45:40.770] PHYSICALDRIVE0 size 512110190592 bytes
[2022/06/28 15:45:40.780] Create file Handle:00000000000000E0 \\.\PhysicalDrive1 status:0
[2022/06/28 15:45:40.789] PHYSICALDRIVE1 size 1000204886016 bytes
```

```
1.0.76 log[2022/06/28 16:00:15.487] Create file Handle:FFFFFFFF \\.\PhysicalDrive0 status:32
[2022/06/28 16:00:15.493] Create file Handle:FFFFFFFF \\.\PhysicalDrive1 status:32
[2022/06/28 16:00:15.502] Create file Handle:00000120 \\.\PhysicalDrive2 status:0
```

**Comment by marwenius** [comment]
Something strange happened: After a reboot also version 1.0.76 does not work anymore. Or earlier versions (tested until 1.0.74).

However, in a live environment (PEBakery/WinPE) it works. It must have something to do with my installation.

Do you have any idea what this could be, i.e. where the status 32 could come from?

**Comment by steve6375** [comment]
Antivirus could be blocking access? Check Services in Task Manager.
Also try using LockHunter to see what processes are locking the USB drive.

**Comment by marwenius** [comment]
Thanks for pointing out the program. However, it is not about external drives, but about internal drives (SSD). I have installed Ventoy on my second SSD.

When I use LockHunter to examine my second SSD, there is only one process (explorer.exe) that shows up and I can unlock it. But even then Ventoy does not work as expected.

I started Windows in safe mode and it worked there. Now I am back in normal mode and it still does not work.

**Comment by steve6375** [comment]
You need to look through services in task manager and look for antivirus programs.
For instance acronis software often installs their antivirus software without you knowing!

**Comment by marwenius** [comment]
I have found the cause: DSAService.exe ([Intel® Driver & Support Assistant](https://www.intel.com/content/www/us/en/support/detect.html)).

As soon as I stopped the process in the task manager, Ventoy worked again. (I simply closed processes until it worked.)

I have now uninstalled the program. Since then, the problem no longer exists.

Thanks for the support! And for Ventoy in general.

![grafik](https://user-images.githubusercontent.com/108332626/176641899-a5ec454f-3c72-4be7-a4b1-db56da3d261e.png)
